core/src/main/java/hudson/Util.java

```diff
@@ -1856,6 +1858,20 @@ public static long daysElapsedSince(@NonNull Date date) {
         return t;
     }
 
+    @Restricted(NoExternalUse.class)
+    public static void printRedirect(String contextPath, String redirectUrl, String message, PrintWriter out) {
+        out.printf(
+                "<html><head>" +
+                "<meta http-equiv='refresh' content='1;url=%1$s'/>" +
+                "<script id='redirect' data-redirect-url='%1$s' src='" +
+                contextPath + Jenkins.RESOURCE_PATH +
+                "/scripts/redirect.js'/>" +
```

Comment:
```suggestion
                "/scripts/redirect.js'></script>" +
```

Reply:
> I'm not sure why

https://html.spec.whatwg.org/#elements-2

`script` is a **raw text element**, and

> [Raw text](https://html.spec.whatwg.org/#raw-text-elements) … elements have a [start tag](https://html.spec.whatwg.org/#syntax-start-tag) to indicate where they begin, and an [end tag](https://html.spec.whatwg.org/#syntax-end-tag) to indicate where they end. The start and end tags of certain [other tags] can be [omitted](https://html.spec.whatwg.org/#syntax-tag-omission) …. Those that cannot be omitted must not be omitted.